docs/resources/cluster_node_pool.md

```diff
@@ -110,6 +114,22 @@ Required:
 - `class` (String) Nodepool instance type
 - `storage_class` (String) Storage Class to be used for storage of the disks which store the root filesystem of the nodes
 
+Optional:
+
+- `failure_domain` (String) Configure the failure domain of node pool. The potential values could be found using cluster:options api. This parameter will be ignored by the backend if the TKG service vsphere cluster doesn't support.
```

Comment:
I think we can rephrase the portion as it feels ambiguous. 
`This parameter will be ignored by the backend if the TKG service vsphere cluster doesn't support.`

Suggest instead:
> Failure domains, also known as vSphere Fault Domains, are defined and managed by the vSphere administrator when creating vSphere Zones. And the parameter will be ignored if failure domains doesn't exist or enabled in vSphere.